flytekit/types/structured/utils.py

```diff
@@ -0,0 +1,37 @@
+import os
+import typing
+
+from flytekit.configuration import aws as _aws_config
+from flytekit.core.data_persistence import split_protocol
+
+
+def get_filesystem(path: typing.Union[str, os.PathLike]) -> typing.Optional["fsspec.AbstractFileSystem"]:
+    protocol, _ = split_protocol(path)
+    if protocol == "s3":
+        kwargs = _get_s3_config()
+        import fsspec
```

Comment:
let's get rid of this for the time being now also, and bring it back as a plugin after?